Formula/swift-format.rb

```diff
@@ -22,10 +22,21 @@ class SwiftFormat < Formula
   # out of the box on Xcode-only systems due to an incorrect sysroot.
   pour_bottle? only_if: :clt_installed
 
-  depends_on xcode: ["12.5", :build]
+  if MacOS.version >= :big_sur
+    depends_on xcode: ["12.5", :build]
+  else
+    depends_on "swift"
+  end
 
   def install
-    system "swift", "build", "--disable-sandbox", "-c", "release"
+    swift = if OS.mac? && MacOS.version <= :catalina
+      f = Formula["swift"]
+      f.opt_prefix/"Swift-#{f.version.major_minor}.xctoolchain/usr/bin/swift"
```

Comment:
Sorry this should be:

```suggestion
      f.opt_prefix/"Swift-#{f.version}.xctoolchain/usr/bin/swift"
```